subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/JUnitSuiteFilteringIntegrationTest.groovy

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.gradle.testing.junit
+
+import org.gradle.integtests.fixtures.DefaultTestExecutionResult
+import org.gradle.integtests.fixtures.MultiVersionIntegrationSpec
+import org.gradle.integtests.fixtures.TargetCoverage
+import org.gradle.testing.fixture.JUnitCoverage
+
+@TargetCoverage({JUnitCoverage.LARGE_COVERAGE})
+public class JUnitSuiteFilteringIntegrationTest extends MultiVersionIntegrationSpec {
```

Comment:
There already is a class called `JunitFilteringIntegrationTest`. I think this should go there too, instead of starting a new one.